modules/mobile/app/controllers/mobile/v0/community_care_providers_controller.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class CommunityCareProvidersController < ApplicationController
+      DEFAULT_PAGE_NUMBER = 1
+      DEFAULT_PAGE_SIZE = 10
```

Comment:
The web app uses 15 and we may change to 15 to match the web app. I don't believe the web app actually allows pagination, so I think the first 15 are the only options available. The mocks don't show any pagination occurring on mobile either. Nonetheless, I've paginated this controller to stay consistent with the upstream service (which is paginated) and allow us to add pagination if we want to.

Reply:
We've agreed on 10.